src/sqlancer/cockroachdb/CockroachDBOracleFactory.java

```diff
@@ -97,7 +100,33 @@ public TestOracle<CockroachDBProvider.CockroachDBGlobalState> create(
         @Override
         public TestOracle<CockroachDBProvider.CockroachDBGlobalState> create(
                 CockroachDBProvider.CockroachDBGlobalState globalState) throws SQLException {
-            return new CockroachDBCERTOracle(globalState);
+            CockroachDBExpressionGenerator gen = new CockroachDBExpressionGenerator(globalState);
+            ExpectedErrors expectedErrors = ExpectedErrors.newErrors().with(CockroachDBErrors.getExpressionErrors())
+                    .build();
+            CERTOracle.CheckedFunction<SQLancerResultSet, Optional<Long>> rowCountParser = (rs) -> {
+                String content = rs.getString(1);
+                if (content.contains("count:")) {
+                    try {
+                        long number = Long.parseLong(content.split("count: ")[1].split(" ")[0].replace(",", ""));
+                        return Optional.of(number);
+                    } catch (Exception e) { // To avoid the situation that no number is found
+                    }
+                }
+                return Optional.empty();
+            };
+            CERTOracle.CheckedFunction<SQLancerResultSet, Optional<String>> queryPlanParser = (rs) -> {
+                String content = rs.getString(1);
+                if (content.contains("• ")) {
+                    String operation = content.split("• ")[1].split(" ")[0];
```

Comment:
The parsing logic for the query plan seems quite brittle. At some point, I guess we should extract these to a method and then have tests that check whether the logic still works when the database systems are updated.